.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,66 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: Please visit our slack chanel [slack](https://adminjs.page.link/slack) or leave your email.
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "List what you are trying to do?"
+    validations:
+      required: true
+  - type: input
+    id: prevalence
+    attributes:
+      label: Bug prevalence
+      description: "How often do you or others encounter this bug?"
+      placeholder: "Example: Whenever I visit the personal account page (1-2 times a week)"
+    validations:
+      required: true
+  - type: textarea
+    id: versions
+    attributes:
+      label: AdminJS dependencies version
+      description: "Provide to us list of dependencies from package.json "
```

Comment:
`Provide to us list of dependencies` -> `Please provide a list of dependencies ...`